### ISSUE TITLE
Low validation rate

### ISSUE THREAD

**Issue by MaartenBloemen** [davidsandberg/facenet#323]
When I try Classifier training of inception resnet v1 as described on the Wiki my validation rate is always very low (max of about 0.5). I have tried several things, I tried upping the epochs, upping the batch size but nothing seems to work, some things even make it worse.
I'm training on the cleaned Casia dataset and get an accuracy of around 94% (which should be 98%) and a Validation rate of 0.52467 ([Full log output](https://pastebin.com/nisUSVKw)), my graphics card is a 1080TI so that is not the issue either.
And I ran using the exact command from the wiki:
`python src/train_softmax.py --logs_base_dir ~/Documents/Logs/facenet/ --models_base_dir ~/Documents/Models/facenet/ --data_dir ~/Documents/Datasets/casia_clean_aligned/ --image_size 160 --model_def models.inception_resnet_v1 --lfw_dir /home/maarten/Documents/Datasets/lfw_aligned_160/ --optimizer RMSPROP --learning_rate -1 --max_nrof_epochs 80 --keep_probability 0.8 --random_crop --random_flip --learning_rate_schedule_file data/learning_rate_schedule_classifier_casia.txt --weight_decay 5e-5 --center_loss_factor 1e-2 --center_loss_alfa 0.9`

Anyone who has an idea of what I'm doing wrong, cause at this point I don't know what to try any more.
Any help I can get at this point would be great, thanks in advance.

**Comment by vudung45** [comment]
Hey man, just out of curiosity, can you please upload a preprocessed picture of yours to imgur for me?

Thank you

**Comment by MaartenBloemen** [comment]
@vudung45 what do you mean by preprocessed? You mean an image that has been aligned so the training can work with it?

**Comment by vudung45** [comment]
@MaartenBloemen yep an aligned image

**Comment by MaartenBloemen** [comment]
In that case sure! [Here you go!](http://imgur.com/a/jRBTM)

**Comment by vudung45** [comment]
I'm not sure if this assumption is correct, but would cropping then resize like this affect the aspect ratio of the face?

**Comment by MaartenBloemen** [comment]
I followed the the wiki so I guess that shouldn't be a problem

**Comment by vudung45** [comment]
@MaartenBloemen try to continue training, start at epoch 80 and ends at like 200 to see if it improves or not

**Comment by MaartenBloemen** [comment]
@vudung45 I already tried that, Ik let the training run for 150 epochs but the validation rate didn't het any better and the accuracy even got worse :(

**Comment by vudung45** [comment]
@MaartenBloemen  You can try to use the pretrained model to speed up ur project for now, then come back trying to reproduce it urself later with this issue got resolved. I suppose it has something to do with the preprocess part.

**Comment by amalF** [comment]
Hello @MaartenBloemen, 

Did you find the answer for your problem ?

**Comment by MaartenBloemen** [comment]
@amalF I recloned the facenet repo and now I get good validation rates. I have no clue what was wrong though...

**Comment by amalF** [comment]
@MaartenBloemen : Thx for answering 
@davidsandberg : Do you an explanation for this issue ? 

Thx

**Comment by hardfish82** [comment]
@davidsandberg I met the same problem while I trained with cleaned CASIA-WebFace.
147000	0.98836	0.54575
148000	0.99111	0.51776
149000	0.98743	0.51442
150000	0.98835	0.50176
151000	0.99111	0.55077
152000	0.98805	0.52811
153000	0.99234	0.54576
154000	0.98713	0.59379
155000	0.99050	0.53370
156000	0.99081	0.56610
157000	0.98805	0.50676
158000	0.99111	0.49439
159000	0.99050	0.49437
160000	0.99295	0.48241
161000	0.98866	0.51306
162000	0.99081	0.50307
163000	0.99142	0.58810
164000	0.99111	0.51876
165000	0.99050	0.56877
166000	0.99111	0.49674
167000	0.99019	0.56575
168000	0.98774	0.52109
169000	0.99081	0.49606
170000	0.98805	0.42065
171000	0.98897	0.56072
172000	0.99081	0.50106
173000	0.99111	0.52943
174000	0.99142	0.52873
175000	0.98989	0.56539
176000	0.98958	0.55379
177000	0.99019	0.49939
178000	0.99173	0.58075
179000	0.99203	0.55809
180000	0.98928	0.57309
181000	0.99019	0.54675
182000	0.98989	0.58379
183000	0.98927	0.52642
184000	0.98927	0.52642
185000	0.99050	0.58543
186000	0.98866	0.54109
187000	0.98743	0.60877
188000	0.98958	0.54974
189000	0.99142	0.55142
190000	0.98989	0.49273
191000	0.98958	0.51608
192000	0.98713	0.56106
193000	0.99111	0.49303
194000	0.99203	0.49941
195000	0.99234	0.48839
196000	0.99050	0.49139
197000	0.99172	0.55609
198000	0.99111	0.48173
199000	0.99142	0.50940
200000	0.99081	0.46771
201000	0.98805	0.56842
202000	0.98988	0.50506
203000	0.98866	0.46508
204000	0.99019	0.45672
205000	0.99050	0.57112
206000	0.99019	0.48504
207000	0.99111	0.54573
208000	0.98989	0.54675
209000	0.99019	0.54843
210000	0.99019	0.53641
211000	0.98989	0.55176
212000	0.99050	0.52477
213000	0.99111	0.50307
214000	0.99019	0.52242
215000	0.99050	0.36360
216000	0.98835	0.51178
217000	0.98743	0.52640
218000	0.99050	0.54609
219000	0.98958	0.50207
220000	0.98774	0.52378
221000	0.98651	0.47972
222000	0.98927	0.51141
223000	0.98713	0.50709
224000	0.98590	0.53374
225000	0.98866	0.52707
226000	0.98713	0.45221
227000	0.98988	0.51607
228000	0.99050	0.51406
229000	0.98805	0.51840
230000	0.98927	0.50475
231000	0.99019	0.49575
232000	0.98866	0.44537
233000	0.98866	0.46638
234000	0.98652	0.49576
235000	0.98835	0.50008
236000	0.99081	0.53007
237000	0.98804	0.47940
238000	0.98927	0.53078
239000	0.98897	0.49806
240000	0.98774	0.52708
241000	0.98927	0.54141
242000	0.98896	0.49398
243000	0.98927	0.50440
244000	0.98835	0.53208
245000	0.98713	0.53611
246000	0.98743	0.47742
247000	0.98866	0.48609
248000	0.98774	0.52444
249000	0.98805	0.49540
250000	0.98958	0.46708
251000	0.98927	0.48776
252000	0.98927	0.46772
